holoviews/element/graphs.py

```diff
@@ -34,13 +34,65 @@ def __call__(self, specs=None, **dimensions):
 
 
 def circular_layout(nodes):
+    """
+    Lay out nodes on a circle and add node index.
+    """
     N = len(nodes)
     circ = np.pi/N*np.arange(N)*2
     x = np.cos(circ)
     y = np.sin(circ)
     return (x, y, nodes)
 
 
+def connect_edges_pd(graph):
```

Comment:
Maybe move some of these functions to element.util?

Reply:
Yes, sounds good.